plugins/collections/README.md

```diff
@@ -53,3 +53,23 @@ Example payload:
     "pipelineIds": [12, 35, 47, 89]
 }
 ```
+
+#### Update a collection
+You can update the name, description, or pipelineIds of a collection.
+
+`PUT /collections/{id}`
+
+**Arguments**
+
+* `name` - An optional new name for the collection. Names must be unique.
+* `description` - An optional new description for the collection.
+* `pipelineIds` - An optional new array of ids of pipelines to be added to the collection.
```

Comment:
We could expose a POST endpoint later on to do additive updates to pipelineIds. When we have a user interaction where user is adding single/multiple pipelines to an existing collection, then an additive POST end point makes sense rather than use this PUT endpoint to send all data each time.